notebooks/401-object-detection-webcam/401-object-detection.ipynb

```diff
@@ -21,22 +21,20 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.                          f"--compress_to_fp16 {True if precision == 'FP16' else False} " \
</pre><p>not sure that it is work, probably you should be:</p><p>if precison == "FP16":</p><p>convert_command + " -- compress_to_fp16"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/828/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Tested. It works as expected